class/defaults.yml

```diff
@@ -22,6 +22,13 @@ parameters:
       cert:
         certManager:
           autoConfigure: true
+      developer:
+        extraOpts:
+          # Force limit the size of heap memory consumption for java based containers in v1.5.0
+          # The value is applied to all containers of a pod in a PostgreSQL instance.
+          # This is a workaround for Stackgres v1.5.0 due to OOM issues in cluster-controller containers.
+          # Bug report - https://gitlab.com/ongresinc/stackgres/-/issues/2452
+          - -XX:MaxHeapSize=512m
```

Comment:
Do we really need such a high size though? AFAIK only the `cluster-controller` is using java in the pod. Or does this also apply to the operator itself?

Reply:
This also applies to stackgres operator.